ListableUI/Sources/HeaderFooter/HeaderFooter.swift

```diff
@@ -56,6 +61,8 @@ public struct HeaderFooter<Content:HeaderFooterContent> : AnyHeaderFooter
         self.sizing = sizing ?? defaults.sizing ?? .thatFits(.noConstraint)
         self.layouts = layouts ?? defaults.layouts ?? .init()
         self.onTap = onTap ?? defaults.onTap
+        self.onEndDisplay = onEndDisplay ?? defaults.onEndDisplay
+        self.onDisplay = onDisplay ?? defaults.onDisplay
```

Comment:
Small tweak to flip the assignment order to match the ordering used in other places:
```suggestion
        self.onDisplay = onDisplay ?? defaults.onDisplay
        self.onEndDisplay = onEndDisplay ?? defaults.onEndDisplay
```

Reply:
Thanks!